python/ray/air/_internal/torch_utils.py

```diff
@@ -10,11 +10,10 @@
 from ray.air.util.data_batch_conversion import _unwrap_ndarray_object_type_if_needed
 
 
-def get_device() -> Union[torch.device, List[torch.device]]:
+def get_devices() -> List[torch.device]:
     """Gets the correct torch device configured for this process.
 
-    Returns a list of devices if more than 1 GPU per worker
-    is requested.
+    Returns a list of devices allocated for the current worker.
```

Comment:
"This is a list of CUDA devices if the current worker has been assigned GPU resources. If no GPUs are assigned, then this returns a list with a single CPU device."

Reply:
Can we also add an example showing before and after:

```python
model.to("cuda")  # Use the GPU that Ray Train sets instead

# vs.

device = ray.train.torch.get_devices()[0]
model.to(device)
```

Reply:
Also, note for the future: we should make sure this method works well outside of ray train so it's easy to run the train fn directly. (e.g. it returns `[torch.device("cuda")] or [torch.device("cpu")]` by default).

Reply:
Agree. Adding examples would help a lot.

Reply:
To make it works outside of ray train, we need to identify if it's in a ray task / actor.